### ISSUE TITLE
Release 0.11.6

### ISSUE THREAD

**Issue by enolfc** [EGI-Foundation/cloud-info-provider#179]
# Short Description of the issue

Prepare and release 0.11.6

**Comment by gwarf** [comment]
* Changes since last version: https://github.com/EGI-Foundation/cloud-info-provider/compare/0.11.5...master

Changelog
```
* Refactor SSL checks (#175) (Enol Fernández)
* Do not fail if JSON cannot be decoded (#176) (Enol Fernández)
* Code Clean-up (#177) (Enol Fernández)
* Import STDOUT and AMS publishers (#178) (Enol Fernández)
```

Release Notes
* Multiple output plugin are now available (STDOUT and Argo Messaging System)
* OpenNebula sites have to ensure that they publish both to BDII and AMS
  * an update of the configuration and parameters is required as documented in the README.md

**Comment by enolfc** [comment]
> Multiple output plugin are now available (STDOUT and Argo Messaging System)
I would use:
* Multiple output plugins are now available (`stdout` for standard output and `ams` for Argo Messaging System)

**Comment by gwarf** [comment]
Fixed by #181
